src/sentry/sentry_metrics/multiprocess.py

```diff
@@ -0,0 +1,334 @@
+import functools
+import logging
+import time
+from typing import (
+    Any,
+    Callable,
+    Dict,
+    Generic,
+    List,
+    Mapping,
+    MutableMapping,
+    Optional,
+    Sequence,
+    TypeVar,
+)
+
+from arroyo.backends.kafka import KafkaConsumer, KafkaPayload, KafkaProducer
+from arroyo.processing import StreamProcessor
+from arroyo.processing.strategies import ProcessingStrategy
+from arroyo.processing.strategies import ProcessingStrategy as ProcessingStep
+from arroyo.processing.strategies import ProcessingStrategyFactory
+from arroyo.processing.strategies.streaming.transform import ParallelTransformStep
+from arroyo.types import Message, Partition, Position, Topic
+from django.conf import settings
+
+from sentry.utils import json, kafka_config
+
+TPayload = TypeVar("TPayload")
+TProcessed = TypeVar("TProcessed")
+
+DEFAULT_QUEUED_MAX_MESSAGE_KBYTES = 50000
+DEFAULT_QUEUED_MIN_MESSAGES = 10000
```

Comment:
Do we have any metrics to measure what we see today or are these the default values we already had ?

Reply:
I just copied them from https://github.com/getsentry/sentry/blob/e582bf7366c72d54d36f1033d31e8fae8e446caf/src/sentry/utils/batching_kafka_consumer.py#L22-L23